data_server/test_data_server.py

```diff
@@ -0,0 +1,82 @@
+import os
+from unittest import mock
+
+import google.cloud.exceptions
+import pytest
+from flask.testing import FlaskClient
+
+from data_server.dataset_cache import DatasetCache
+from main import app, cache
+
+os.environ['GCS_BUCKET'] = 'test'
+os.environ['METADATA_FILENAME'] = 'test_data.ndjson'
+
+test_data = b"""
+{"label1":"value1","label2":["value2a","value2b","value2c"],"label3":"value3"}
+{"label1":"value2","label2":["value3a","value2b","value2c"],"label3":"value6"}
+{"label1":"value3","label2":["value4a","value2b","value2c"],"label3":"value9"}
+{"label1":"value4","label2":["value5a","value2b","value2c"],"label3":"value12"}
+{"label1":"value5","label2":["value6a","value2b","value2c"],"label3":"value15"}
+{"label1":"value6","label2":["value7a","value2b","value2c"],"label3":"value18"}
+"""
+
+
+def get_test_data(gcs_bucket: str, filename: str):
+    """Returns the contents of filename as a bytes object. Meant to be used to
+    patch gcs_utils.download_blob_as_bytes."""
+    print('In get_test_data')
```

Comment:
Should this still be here? Or was it just used for debugging?

Reply:
My bad, this one slipped through